vagrant/scripts/install_iml_repouri.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [[ -z $1 ]];
+then
+    echo "REPO_URI must be set to use this provisioner"
```

Comment:
This can be confusing, because `REPO_URI` is not used anywhere is this script.

Reply:
Technically, environment variables can be passed to scripts => https://www.vagrantup.com/docs/provisioning/shell.html#env

Reply:
`REPO_URI` is what's expected to be set by the user invoking this provisioner.